numba/typed/typedobjectutils.py

```diff
@@ -149,7 +149,7 @@ def _get_incref_decref(context, module, datamodel, container_element_type):
     return incref_fn, decref_fn
 
 
-def _get_equal(context, module, datamodel, container_element_type):
+def _get_container_equal(context, module, datamodel, func_name_suffix):
```

Comment:
You can leave the name as `container_element_type` as it is what it's used in the rest of this file.

Reply:
removed it, since it can be an internal detail.